Remotion/Core/Core/Reflection/TypeDiscovery/AssemblyLoading/RegexAssemblyLoaderFilter.cs

```diff
@@ -87,7 +87,7 @@ public bool ShouldConsiderAssembly (AssemblyName assemblyName)
       switch (_matchTarget)
       {
         case MatchTargetKind.SimpleName:
-          return _matchExpression.IsMatch (assemblyName.Name!); // TODO RM-7770: assemblyName.Name should be checked for null.
+          return _matchExpression.IsMatch (assemblyName.GetNameChecked()); // TODO RM-7770: assemblyName.GetNameSafe() should be checked for null.
```

Comment:
```suggestion
          return _matchExpression.IsMatch (assemblyName.GetNameChecked());
```
link todo ticket and mark as resovled